storage/container_storage_interface/persistent-storage-csi-azure-file.adoc

```diff
@@ -0,0 +1,40 @@
+[id="persistent-storage-csi-azure-file"]
+= Azure File CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-azure-file
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Microsoft Azure File Storage.
+
+:FeatureName: Azure File CSI Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
```

Comment:
Could restructure to avoid passive voice. "Review resource x and resource y before proceeding," or somesuch.

Reply:
Since this phrase is pre-dates me and is in all of the storage topics, I'm leaving it.